### ISSUE TITLE
[BUG] Dark mode CSS: Text is unreadable (black-on-black problem) 

### ISSUE THREAD

**Issue by trinitronx** [JScearcy/rust-doc-viewer#43]
**Describe the bug**
Similar in symptoms to #36: Dark mode is selected but text foreground color is also still black.
So this causes black-on-black readability problems when viewing the doc pages.

**To Reproduce**

1. Install `JScearcy.rust-doc-viewer` version `4.0.0`
2. Open Rust project in VSCode
3. Run `cargo doc` to generate docs
4. Open Command Palette: Press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> <sup>(Linux & Windows)</sup> / <kbd>⌘</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> <sup>(Mac)</sup>
5. Select: `Rust: Doc Viewer`

**Expected behavior**
CSS dark mode styles should be readable, similar to how `doc.rust-lang.org` doc pages look.  (See example below)

**Screenshots**
How `doc.rust-lang.org` looks with dark mode style:
![swappy-20230908_180144](https://github.com/JScearcy/rust-doc-viewer/assets/122524/21556d5a-c94c-4855-b3db-7310ecd904ca)

How rust docs look in VSCode:
![swappy-20230908_181244](https://github.com/JScearcy/rust-doc-viewer/assets/122524/2f0847ef-5df4-4930-ae4b-893a5230d0f3)

**Additional context**
Reproduced on `Manjaro Linux 23.0.0 - 6.1.49-1-MANJARO`

<details><summary><b>EXPAND</b> for platform / package version info</summary>
<p>

```
Version: 1.81.1
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-11T08:12:48.924Z
Electron: 22.3.22
ElectronBuildId: undefined
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Linux x64 6.1.49-1-MANJARO


A/B Experiments:

vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vshan820:30294714
vstes263cf:30335440
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30803845
3biah626:30602489
89544117:30613380
showlangstatbar:30737416
03d35959:30757346
fixshowwlkth:30771522
showindicator:30805244
i26e3531:30792625
asynctok:30821568
dsvsc013:30795093
dsvsc014:30804076
diffeditorv2:30821572
dsvsc015cf:30829746
```

CPUs | Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz (8 x 4400)
-- | --
GPU Status | 2d_canvas: enabled canvas_oop_rasterization: disabled_off direct_rendering_display_compositor: disabled_off_ok gpu_compositing: enabled multiple_raster_threads: enabled_on opengl: enabled_on rasterization: enabled raw_draw: disabled_off_ok video_decode: enabled video_encode: disabled_software vulkan: disabled_off webgl: enabled webgl2: enabled webgpu: disabled_off
Load (avg) | 1, 1, 1
Memory (System) | 62.69GB (54.78GB free)
Process Argv | --ozone-platform-hint=wayland --enable-features=WaylandWindowDecorations --enable-features=WebRTCPipeWireCapturer --force-dark-mode --password-store=gnome-libsecret --unity-launch --enable-proposed-api GitHub.vscode-pull-request-github --enable-proposed-api command line flag --enable-proposed-api ms-python.python
Screen Reader | no
VM | 0%

</p>
</details>

Rust toolchain info:

```sh
$ rustup show

Default host: x86_64-unknown-linux-gnu
rustup home:  ~/.rustup

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu (default)
nightly-x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.72.0 (5680fa18f 2023-08-23)

$ cargo --version
cargo 1.72.0 (103a7ff2e 2023-08-15)
```

Rust-related VSCode Extensions Installed:

Extension | Author (truncated) | Version
-- | -- | --
rust-doc-viewer | JSc | 4.0.0
rust-targets | Pol | 1.1.1
rust-analyzer | rus | 0.4.1653
crates | ser | 0.5.11
vscode-rust-test-adapter | Swe | 0.11.1
even-better-toml | tam | 0.19.1

<details><summary><b>EXPAND</b> for FULL Extensions List</summary>
<p>

Extension | Author (truncated) | Version
-- | -- | --
arepl | alm | 2.0.5
atlascode | atl | 3.0.5
gnome-magic | bed | 0.2.2
flatpak-vscode | bil | 0.0.35
vscode-modelines | chr | 2.0.6
vscode-github-actions | csc | 0.24.4
doxdocgen | csc | 1.4.0
vscode-markdownlint | Dav | 0.49.0
groovy-guru | Don | 0.6.0
gitlens | eam | 14.3.0
simple-alignment | ear | 1.0.1
vscode-pull-request-github | Git | 0.70.0
go | gol | 0.39.1
terraform | has | 2.27.2
test-adapter-converter | hbe | 0.1.6
vscode-test-explorer | hbe | 2.21.1
auto-markdown-toc | hun | 3.0.12
better-cpp-syntax | jef | 1.21.1
rust-doc-viewer | JSc | 4.0.0
vscode-python-test-adapter | lit | 0.7.1
vscode-clangd | llv | 0.1.24
mesonbuild | mes | 1.10.0
python-path | mge | 0.0.14
vscode-docker | ms- | 1.25.2
vscode-dotnet-runtime | ms- | 1.7.2
vscode-kubernetes-tools | ms- | 1.3.13
isort | ms- | 2022.8.0
mypy-type-checker | ms- | 2023.2.0
python | ms- | 2023.14.0
jupyter | ms- | 2023.7.100
jupyter-keymap | ms- | 1.1.2
jupyter-renderers | ms- | 1.0.15
vscode-jupyter-cell-tags | ms- | 0.1.8
cmake-tools | ms- | 1.15.31
hexeditor | ms- | 1.9.9
makefile-tools | ms- | 0.7.0
go-lazy-mock | naw | 1.0.3
pytest-fixtures | nic | 0.3.3
linux-desktop-file | nic | 0.0.21
vscode-groovy-lint | Nic | 2.0.0
rust-targets | Pol | 1.1.1
ruby | reb | 0.28.1
ansible | red | 2.7.98
vscode-xml | red | 0.26.1
vscode-yaml | red | 1.14.0
rust-analyzer | rus | 0.4.1653
scala | sca | 0.5.6
crates | ser | 0.5.11
blueprint-gtk | son | 0.2.0
vscode-rust-test-adapter | Swe | 0.11.1
even-better-toml | tam | 0.19.1
python-tox | the | 1.0.0
cmake | twx | 0.0.17
vscode-choosealicense | ult | 0.9.3
vscode-lldb | vad | 1.9.2
vscode-ruby | win | 0.28.0

<span style="color: rgb(204, 204, 204); font-family: system-ui, Ubuntu, &quot;Droid Sans&quot;, sans-serif; font-size: 12px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(42, 42, 42); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">(1 theme extensions excluded)</span>

</p>
</details>

**Comment by sophie-iren-katz** [comment]
+1 seeing this on macOS with Dark Modern theme.

**Comment by JScearcy** [comment]
This definitely looks similar, if not the same issue as the previous bug. I'm checking into it

**Comment by JScearcy** [comment]
A change in the generated `index.html` from `cargo doc` caused some trouble with paths. I published 4.0.1 which handles both versions of the file, and should resolve this issue

**Comment by trinitronx** [comment]
> I published 4.0.1 which handles both versions of the file, and should resolve this issue

Thanks!  It looks like this fixed the issue for me:

![swappy-20230915_035009](https://github.com/JScearcy/rust-doc-viewer/assets/122524/289288cc-37e9-4a02-8b20-af343babd55a)
